route/book-router.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+// npm dependencies
+const {Router} = require('express');
+const httpErrors = require('http-errors');
+const jsonParser = require('body-parser').json();
+
+// app dependencies
+const Book = require('../model/book.js');
+// module interface
+const bookRouter = module.exports = new Router();
+
+// POST /api/<resource-name>
```

Comment:
Comments are great, but too many of them can be verbose and hard to maintain. Ideally, the code should be understandable without the need of comments.

I'll mark the comments that might be a little bit too much